app/models/user.rb

```diff
@@ -52,6 +52,18 @@ def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
 
+  def create_reset_digest
+    self.reset_token = User.new_token
+    update(reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now)
```

Comment:
```suggestion
    update_columns reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```
chỗ ni tối ưu nhất là dùng method update column nhé vì chỗ này cần j chạy callback hay validate j đâu cu